runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -219,15 +220,15 @@ There are a some words fun that don't belong paper in this sentence.
 
 # Lesson 2.2: MORE DELETION COMMANDS
 
-** Type `d$`{normal} to delete to the end of the line. **
+** Type `d$`{normal} or `D`{normal} to delete to the end of the line. **
```

Comment:
This is a general question (which I don't know the answer to): is knowing two different ways to do the same thing helpful or confusing at this point? (`d$` teaches two things: the command _and_ the motion, as well as the general structure -- if that instruction were expanded, I'd prefer to add an explanation rather than a different command.)

Reply:
I can't think of a good way to word the explanation. I was thinking just add a note under at the end of the lesson, but I'm not sure what to put

However, I have still removed the "or [DCY]" from those lines

Reply:
As to your inital question here however.. I'm not sure that showing both ways is better but I would say learning D is better in the long run than learning d$, I just don't want to take away from teaching what $ means by itself